docs/docs/documentation/getting-started/installation/backend-config.md

```diff
@@ -16,7 +16,8 @@
 | TZ                            |          UTC          | Must be set to get correct date/time on the server                                  |
 | ALLOW_SIGNUP<super>\*</super> |         false         | Allow user sign-up without token                                                    |
 | LOG_CONFIG_OVERRIDE           |                       | Override the config for logging with a custom path                                  |
-| LOG_LEVEL                     |         info          | logging level configured                                                            |
+| LOG_LEVEL                     |         info          | Logging level configured                                                            |
+| DAILY_SCHEDULE_TIME           |        23:45          | The time of day to run the daily tasks.                                |
```

Comment:
Can we extend the last pipe out so it lines up? Just for my own sanity, I don't think it actually matters

Reply:
I'd rather not try to be pedantic about this .. that whole table structure is a bit of a mess, and any change made to try and fix it now will just get "broken" in future. There's a ton of inconsistency in there already.

Reply:
Huh, maybe it's just GitHub then, it seems to be formatted correctly to me:

![image](https://github.com/mealie-recipes/mealie/assets/71845777/98c96d55-d98c-4e41-a926-f75f73fdd4dc)

But whatever it really doesn't matter anyway

Reply:
No, you're correct. VSCode was completely distorting it for me 🤦 

Fixed.